### PR TITLE
[qa] Cleanup isort and flake8 settings in an OpenWISP module #13

### LINKED ISSUE #13: [qa] Cleanup isort and flake8 settings

### LINKED ISSUE #14: [qa] Add message check commit

### PR DESCRIPTION
**Pull request by ppabcd**
Clean isort and flake. Added commit message check in travis.
Fixes #13
Fixes #14

### CODE REVIEW

File: openwisp_ipam/models.py [hunk 1]
```diff
@@ -17,11 +17,11 @@ def request_ip(self, options=None):
...
-        ip_address = load_model('openwisp_ipam', 'IpAddress')(
-                                ip_address=ip,
-                                subnet=self,
-                                organization=self.organization,
-                                **options)
+        ip_address = load_model('openwisp_ipam',
+                                'IpAddress')(ip_address=ip,
+                                             subnet=self,
+                                             organization=self.organization,
+                                             **options)
```
Comment (nemesisdesign) [inline openwisp_ipam/models.py]: revert this change as it's not related to this patch and introducing unrelated changes is a bad practice

Comment (ppabcd) [reply openwisp_ipam/models.py]: i always get errors with flake8 and i modified this files. I'll rollback this file.

**Comment by ppabcd** [comment]
all done @nemesisdesign

**Comment by atb00ker** [comment]
@ppabcd yes please, understand why the formatting in the file is giving error and fix it accordingly, feel free to ask if you get stuck ! :smile:

**Comment by ppabcd** [comment]
all done @atb00ker 
gambar
